app/views/qsets/_modal_edit_qset.html.erb

```diff
@@ -31,9 +31,11 @@
             </div>
             <div class="form-group">
               <%= permissions.label :qset_type, 'Qset type:' %>
-              <%= permissions.label :qset_type_questions, 'Questions' %>
+              <%= permissions.label :qset_type_mixed, 'Mixed', :class => 'qset-type-padding' %>
+              <%= permissions.radio_button :qset_type, 'mixed' %>
+              <%= permissions.label :qset_type_questions, 'Questions', :class => 'qset-type-padding' %>
               <%= permissions.radio_button :qset_type, 'questions' %>
-              <%= permissions.label :qset_type_subsets, 'Subsets' %>
+              <%= permissions.label :qset_type_subsets, 'Subsets', :class => 'qset-type-padding' %>
```

Comment:
FYI there is a more succinct way to do this. e.g. https://github.com/penzance/askup/blob/master/app/views/qsets/_modal_edit_qset.html.erb#L42